data/ka'het outfits.txt

```diff
@@ -0,0 +1,194 @@
+	cost 529000
+	thumbnail "outfit/ka'het annihilator turret"
+	"mass" 25
+	"outfit space" -32
+	"weapon capacity" -32
+	"turret mounts" -1
+	weapon
+		sprite "projectile/annihilator"
+			"frame rate" 7
+		"hardpoint sprite" "hardpoint/annihilator turret"
+		"hardpoint offset" 15.
+		sound "plasma"
+		"hit effect" "bullet impact"
+		"inaccuracy" 3
+		"turret turn" 2.3
+		"velocity" 18
+		"lifetime" 35
+		"reload" 10
+		"firing energy" 62
+		"firing heat" 16
+		"shield damage" 86
+		"hull damage" 24
+	description "The Builders mounted two Annihilators on this turret, making it highly efficent against enemy shields."
+
+outfit "Ka'het EMP Deployer"
+	category "Secondary Weapons"
+	cost 715000
+	thumbnail "outfit/ka'het emp deployer"
+	"mass" 18
+	"outfit space" -26
+	"weapon capacity" -26
+	"gun ports" -1
+	"emp torpedo capacity" 16
+	weapon
+		sprite "projectile/emp torpedo"
+			"frame rate" 2
+			"no repeat"
+		sound "torpedo"
+		ammo "EMP Torpedo"
+		icon "icon/emp"
+		"fire effect" "emp torpedo fire" 5
+		"hit effect" "nuke explosion"
+		"hit effect" "emp ring" 30
+		"hit effect" "emp spark" 20
+		"die effect" "tiny explosion"
+		stream
+		"reload" 180
+		"firing energy" 540
+		"firing heat" 11
+		"inaccuracy" 4
+		"lifetime" 300
+		"velocity" 7
+		"acceleration" .7
+		"drag" .1
+		"turn" 2
+		"homing" 3
+		"infrared tracking" .7
+		"radar tracking" .8
+		"trigger radius" 50
+		"blast radius" 200
+		"shield damage" 700
+		"hull damage" 100
+		"ion damage" 48
+		"hit force" 90
+		"missile strength" 80
+	description "Eons ago, the Builders were attacked by alien ships which used weapons that ignored their hull and damaged their reactors. Although they never did recover any of these weapons, after years of research they managed to replicate their effects."
+
+# Cooling:
+
+outfit "Ka'het Support Cooling"
+	category "Systems"
+	cost 218000
+	thumbnail "outfit/ka'het support cooling"
+	"mass" 6
+	"outfit space" -8
+	"active cooling" 32
+	"cooling energy" 3.1
+	description "Het ships need a much better cooling than normal ships, partly due to the requirement of keeping the Het alive. While a single Support Cooling isn't enough to sustain a ship, the Ka'het use this module in addition to the stronger Primary Cooling."
+
+
+outfit "Ka'het Primary Cooling"
+	category "Systems"
+	cost 563000
+	thumbnail "outfit/ka'het primary cooling"
+	"mass" 11
+	"outfit space" -14
+	"active cooling" 65
+	"cooling energy" 5.9
+	description "This is the main cooler used by the Ka'het. Although power-hungry, it is very effective, capable of upholding some types of Het by itself."
+
+# Shield:
+
+outfit "Ka'het Shield Restorer"
+	category "Systems"
+	cost 138000
+	thumbnail "outfit/ka'het shield restorer"
+	"mass" 15
+	"outfit space" -19
+	"shield generation" 1.02
+	"shield energy" 2.04
+	description "The Builders never relied much on shield matrices, preferring instead to strengthen their hull. This, however, forced some Het vessels to make extensive use of this module."
```

Comment:
This could be characterized better, since the whole "forced" idea doesn't make sense to me, even with the outside explanation. Perhaps change the second sentence entirely to something like "The reality of war required that the Builders develop strong shield technology, though, leading to some Het vessels making extensive use of this module."